src/support/z_sitetest.erl

```diff
@@ -0,0 +1,129 @@
+%% @author Arjan Scherpenisse <arjan@miraclethings.nl>
+%% @copyright 2016 Arjan Scherpenisse
+%%
+%% @doc Run site-specific tests in an isolated database schema
+
+%% Copyright 2016 Arjan Scherpenisse <arjan@miraclethings.nl>
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%% 
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%% 
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(z_sitetest).
+-author("Arjan Scherpenisse <arjan@miraclethings.nl>").
+
+-export([run/1, run/2, find_sitetest_modules/1]).
+
+-include_lib("zotonic/include/zotonic.hrl").
+-include_lib("epgsql/include/pgsql.hrl").
+
+%% @doc Run all _sitetest tests for the given host
+run(Host) when is_atom(Host) ->
+    run(Host, find_sitetest_modules(Host)).
```

Comment:
Yes, let's change this in the master. Consistent naming is important.

So for now on we will be talking about a `site` running on Zotonic. Which is much more consistent with the `sites` directory anyway. I will make a change request for this.